File: test/fixtures/output/ocaml/cohttp/cookies.ml
open Cohttp_lwt_unix
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.init ()
  |> fun h -> Header.add h "Cookie" "foo=bar; bar=baz"
in

Client.call ~headers (Code.method_of_string "POST") uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: src/targets/node/unirest/fixtures/cookies.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

const CookieJar = unirest.jar();
CookieJar.add('foo=bar', 'http://mockbin.com/har');
CookieJar.add('bar=baz', 'http://mockbin.com/har');
req.jar(CookieJar);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});